Patches/Vanilla Factions Expanded - Empire/Ammo/FletcherRifleRound.xml

```diff
@@ -48,6 +48,7 @@
 								<MarketValue>6.78</MarketValue>
 							</statBases>
 							<ammoClass>Fletcher</ammoClass>
+							<isMortarAmmo>true</isMortarAmmo>
```

Comment:
What's the reason for this line? From what I've seen and tested it just lets you order the ammo from the deserters table with the ammo mode disabled.

Reply:
Removed it. You're correct--I for some reason thought it kept the ammo in Generic Ammo mode, when in fact it only does that in No Ammo mode, which isn't helpful to is.